pkg/cmdlets/errors/errors.go

```diff
@@ -85,6 +85,21 @@ func trimError(msg string) string {
 	return strings.TrimRight(msg, " .")
 }
 
+func (o *OutputError) MarshalStructured(f output.Format) interface{} {
```

Comment:
Unwrap all errors and return a list of error strings to be converted to a `{"errors":["error1","error2",...],"code":x}` JSON object.